### PR TITLE
PROGRAMMES-6765 - Good luck reviewing this hahahahahahahahahahahahahaha

### PR DESCRIPTION
**Pull request by Bobkidbob**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6765

### CODE REVIEW

**Comment by cisco87** [comment]
So the first time we duplicated all over the world `spacer-hidden` because it would never change, then in less than a week we had to change it all over the world, but I guess creating constants to change repeating values everywhere easily is too mainstream? :D

**Comment by cisco87** [comment]
I would advise to remove the integers around the code base and show mercy towards the one who will have to squeeze his wits to understand what each integer represents

File: src/Ds2013/Presenters/Utilities/Paginator/PaginatorPresenter.php [hunk 1]
```diff
@@ -59,6 +59,13 @@ public function shouldApplyNoHideClass(int $item): bool
         return $this->currentPageWithinThreeOfEnd($item);
     }
 
+    public function shouldApplyMobileClass(int $item): bool
+    {
+        $pages = $this->getPageCount();
+        $currentPage = $this->currentPage;
+        return ($item == 2 || $item == $pages - 1) && ($currentPage == 5 || $currentPage == $pages - 4);
```
Comment (Tasiobg) [inline src/Ds2013/Presenters/Utilities/Paginator/PaginatorPresenter.php]: I think is a good practice to use the Identical operator `===` when possible

File: src/Ds2013/Presenters/Utilities/Paginator/PaginatorPresenter.php [hunk 2]
```diff
@@ -59,6 +59,13 @@ public function shouldApplyNoHideClass(int $item): bool
         return $this->currentPageWithinThreeOfEnd($item);
     }
 
+    public function shouldApplyMobileClass(int $item): bool
```
Comment (Tasiobg) [inline src/Ds2013/Presenters/Utilities/Paginator/PaginatorPresenter.php]: Some comments for this function could help, Is not clear why it should apply mobile class

**Pull request merged**